backend/server/acl.go

```diff
@@ -188,6 +188,48 @@ func enforceWorkspaceDeveloperSheetRouteACL(plan api.PlanType, path string, meth
 	return nil
 }
 
+var issueStatusRegex = regexp.MustCompile(`^/issue/(?P<issueID>\d+)/status`)
+
+// enforceIssueRouteACL enforces the ACL for /issue route, return true if the request is allowed, error is not nil indicates encounter error during the process.
+func enforceIssueRouteACL(path string, method string, queryParams url.Values, principalID int, role api.Role, isWorkspaceDeveloperMemberOfIssue func(issueID int, principalID int) error) *echo.HTTPError {
+	if !strings.HasPrefix(path, "/issue") {
+		return nil
+	}
+
+	if method == "GET" {
+		// For /issue route, users excluding BytebaseBot cannot list other users' issues.
```

Comment:
```suggestion
		// For /issue route, users other than BytebaseBot cannot list other users' issues.
```

Reply:
We don't have private project. So issues are all public. I don't feel we need to restrict read access.

Also, at least, workspace owner and dba should be able to view all issues.

Reply:
All issues should be available to workspace owner/dba's who have admin privileges. We can restrict access to project owners for now because SQL statements (DMLs) are somewhat private within the project members.

Reply:
Fine with me

Reply:
@h3n4l need to allow owner and dba per discussion

Reply:
Only workspace developer will enter this logic.